lastpass-cli/plan.sh

```diff
@@ -0,0 +1,49 @@
+pkg_name=lastpass-cli
+pkg_origin=core
+pkg_version=1.3.3
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("GPL-2.0-or-later")
+pkg_description="LastPass CLI is a utility to progromatically interact with LastPass"
+pkg_upstream_url="https://github.com/lastpass/lastpass-cli"
+pkg_source="https://github.com/lastpass/lastpass-cli/releases/download/v${pkg_version}/${pkg_name}-${pkg_version}.tar.gz"
+pkg_filename="${pkg_name}-${pkg_version}.tar.gz"
```

Comment:
pkg_filename here should not be necessary, as (if removed) it will be the default. Just a note / FYI.